tooling/reproducible/linux_repro_build_compare.sh

```diff
@@ -84,8 +84,116 @@ setEnvironment() {
   ls -ld "$CC" "$CXX" "/usr/lib/jvm/jdk-${BOOTJDK_VERSION}/bin/javac" || exit 1
```

Comment:
split this method into separate:
setGccEnvironment()
setAntEnvironment()
Also remove check on javac, as it might not be located in /usr/lib/jvm/jdk-